docs/book/src/developer/e2e.md

```diff
@@ -58,25 +58,25 @@ Using the config file it is possible to:
   `clusterctl config cluster`.
 - Define a list of intervals to be used in the test specs for defining timeouts for the
   wait and `Eventually` methods.
-- Define the list of images to be loaded in the management cluster (this is specif of
-  management cluster based on kind).
+- Define the list of images to be loaded in the management cluster (this is specific to
+  management clusters based on kind).
 
 An [example E2E config file] can be found here.
 
 <aside class="note">
 
 <h1>Deprecated E2E config file format</h1>
 
-The [Cluster API test framework] includes also a [deprecated E2E config file] implementation,
-that was used before the introduction of clusterctl. This might be removed in future releases 
+The [Cluster API test framework] also includes a [deprecated E2E config file] implementation,
```

Comment:
AFAIK this has been removed in v1alpha4, so we can get rid of this

Reply:
Great, I will remote this section.